index.Rmd

```diff
@@ -701,142 +698,63 @@ Refer to the reference section of this document for further information on funct
 
 ```{block, type="alert alert-info"}
 # (optional) Writing packages
-The next step to write re-usable code is packaging it, so others can simply install and use it. If you followed the steps to here, this step is not very big anymore! For this course, it is optional. Find instructions [here](https://r-pkgs.org/intro.html) and in the [references](#References_and_more_info) below.
+The next step to write re-usable code is packaging it, so others can simply install and use it. If you followed the steps to here, this step is not very big anymore! For this course, it is optional. Find instructions [here](https://r-pkgs.org/introduction.html) and in the [references](#References_and_more_info) below.
 ```
 
 # Creating a map within *R* - a simple demo
 
-Here is an example of how you can create a map in *R*. It is a function to get *public* data from anywhere in the world. See help of the `getData` function in the `raster` package.
+Here is an example of how you can create a map in *R*. We will make use of a subset of the [Global Adminstrative Areas database (GADM)](http://www.gadm.org):
 
+```{r}
+# Create data and output folders and download data from URL
+data_URL <- "https://github.com/GeoScripting-WUR/Scripting4GeoIntro/releases/download/gadm-data/gadm41_PHL_2.zip"
+data_folder <- "data"
```

Comment:
Note that they are called "folders" only in Windows. On Linux they are called "directories". So perhaps `data_dir` would be a more cross-platform name.